cmd/ponzu/usage.go

```diff
@@ -162,6 +162,19 @@ var usageVersion = `
 
 `
 
+var usageAdd = `
+[--cli] add, a <addon URI>
```

Comment:
is `[--cli]` actually used in the add command? if not, lets clear that so it's not confusing. then in place of <addon URI>, let's say <import-path>